packages/@aws-cdk/toolkit-lib/build-tools/package.sh

```diff
@@ -10,7 +10,9 @@ cd "$(dirname $(dirname "$0"))"
 rm -rf dist/
 mkdir -p dist/js
 npm pkg set version=0.0.0-alpha.$commit
+npm pkg set dependencies.@aws-cdk/cloud-assembly-schema=$version
 npm pkg set dependencies.@aws-cdk/cloudformation-diff=$version
 npm pack --pack-destination dist/js
 npm pkg set version=$reset
-npm pkg set dependencies.@aws-cdk/cloudformation-diff=^$reset
+npm pkg set dependencies.@aws-cdk/cloudformation-diff=^$reset``
+npm pkg set dependencies.@aws-cdk/cloud-assembly-schema=^$reset``
```

Comment:
are these "`" intentional? never seen before but im not good at shell scripting

Reply:
no. thanks for catching!